### PR TITLE
Migrate client id

### PR DESCRIPTION
**Pull request by llbartekll**
# Description
update client id `kSecAttrAccessible` to `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly`, so it can be accessed when app is in background. - potential fix for a crash.

## How Has This Been Tested?
manually:
1. build app with client id `kSecAttrAccessibleWhenUnlockedThisDeviceOnly` as on production
2. client Id keychain item is updated after running with the following changes

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Example/ExampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2572,11 +2572,9 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
 				CODE_SIGN_ENTITLEMENTS = PNDecryptionService/PNDecryptionService.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
```
Comment (flypaper0) [inline Example/ExampleApp.xcodeproj/project.pbxproj:2575]: its for DEBUG configuration only, right?

Comment (llbartekll) [reply Example/ExampleApp.xcodeproj/project.pbxproj:2575]: yes

File: Sources/WalletConnectRelay/RelayClientFactory.swift [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
+import Foundation
+
+
+public struct RelayClientFactory {
```
Comment (flypaper0) [inline Sources/WalletConnectRelay/RelayClientFactory.swift:5]: 👍

**Review by flypaper0 — APPROVED** [review]

**Review by flypaper0 — APPROVED** [review]

**Comment by llbartekll** [comment]
Closing for now as [this](https://github.com/WalletConnect/WalletConnectSwiftV2/pull/896) may fix the issue and migration may be not needed.

**Pull request closed without merging**